packages/flutter_tools/lib/src/web/file_generators/flutter_js.dart

```diff
@@ -205,6 +205,8 @@ _flutter.loader = null;
         this._didCreateEngineInitializerResolve(engineInitializer);
         // Remove the resolver after the first time, so Flutter Web can hot restart.
         this._didCreateEngineInitializerResolve = null;
+        // Make the engine revert to "auto" initialization on hot restart.
+        // delete _flutter.loader.didCreateEngineInitializer;
```

Comment:
LGTM, once this line is uncommented :)

Reply:
Yes, I want to see this fail, then pass in CI.

I've run the tests locally, but I want to make sure that I'm not breaking anything weird.